test/noop/noop_test.rb

```diff
@@ -87,8 +87,7 @@ def test_padrino_not_instrumented
   end
 
   def test_graphql_not_instrumented
-    refute GraphQL::Schema.plugins.find { |plugin| plugin == GraphQL::Tracing::AppOpticsTracing },
-           'failed: This should be noop mode, but GraphQL is instrumented.'
+    refute defined?(GraphQL::Tracing::SolarWindsAPMTracing) && GraphQL::Schema.singleton_class.includes?(GraphQL::Tracing::SolarWindsAPMTracingrwindsAPM)
```

Comment:
Minor: I think this is a typo: `SolarWindsAPMTracingrwindsAPM` --> `SolarWindsAPMTracing`